sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/ChaosFaultResourceData.cs

```diff
@@ -0,0 +1,96 @@
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson("foo")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        private IDictionary<string, BinaryData> _serializedAdditionalRawData;
+
+        /// <summary> Initializes a new instance of <see cref="ChaosFaultResourceData"/>. </summary>
+        public ChaosFaultResourceData()
+        {
+        }
+
+        /// <summary> Initializes a new instance of <see cref="ChaosFaultResourceData"/>. </summary>
+        /// <param name="id"> The id. </param>
+        /// <param name="name"> The name. </param>
+        /// <param name="resourceType"> The resourceType. </param>
+        /// <param name="systemData"> The systemData. </param>
+        /// <param name="action"> Indicates whether what action to take for the Chaos Fault. </param>
+        /// <param name="region"> Region of the account where the Chaos Fault is to be enabled/disabled. </param>
+        /// <param name="databaseName"> Database name. </param>
+        /// <param name="containerName"> Container name. </param>
+        /// <param name="provisioningState"> A provisioning state of the Chaos Fault. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal ChaosFaultResourceData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, SupportedAction? action, string region, string databaseName, string containerName, string provisioningState, IDictionary<string, BinaryData> serializedAdditionalRawData) : base(id, name, resourceType, systemData)
+        {
+            Action = action;
+            Region = region;
+            DatabaseName = databaseName;
+            ContainerName = containerName;
+            ProvisioningState = provisioningState;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Indicates whether what action to take for the Chaos Fault. </summary>
+        [WirePath("properties.action")]
+        public SupportedAction? Action { get; set; }
+        /// <summary> Region of the account where the Chaos Fault is to be enabled/disabled. </summary>
+        [WirePath("properties.region")]
+        public string Region { get; set; }
```

Comment:
Just want to check. Is this actually an `AzureLocation`?

Reply:
It is, but we're consuming it as a string in our API for the time being to test out different scenarios. Will update in next version if needed.